.circleci/config.yml

```diff
@@ -3,21 +3,28 @@ version: 2
 jobs:
   build:
     docker:
-      - image: sensirion/embedded-ci:2.3.0
+      - image: sensirion/embedded-ci:2.4.0
     steps:
       - checkout
       - run: make test
 
   style-check:
     docker:
-      - image: sensirion/embedded-ci:2.3.0
+      - image: sensirion/embedded-ci:2.4.0
     steps:
       - checkout
       - run: make style-check
 
+  run-cppcheck:
+    docker:
+      - image: sensirion/embedded-ci:2.4.0
+    steps:
+      - checkout
+      - run: cppcheck --std=c++11 --language=c++ --error-exitcode=1 --enable=warning,style,performance,portability -i i2c/sample-implementations/ -i shdlc/sample-implementations/ .
```

Comment:
It's not c++, so I'd change the language to `c` and the std to `c89` or `c99`. I'm not sure if we should try to support `c89` for really old embedded compiler or just go for `c99`.

Reply:
Choose `c99`.